src/backend/distributed/utils/node_metadata.c

```diff
@@ -524,12 +538,31 @@ master_update_node(PG_FUNCTION_ARGS)
 	 */
```

Comment:
The comment above might need to be rephrased a bit, mainly the drawback.

Reply:
I have added some extra information here, could you have a look if that is what you were looking for?